src/executables/compactdb_cl.c

```diff
@@ -831,7 +831,7 @@ compactdb_start (bool verbose_flag, bool delete_old_repr_flag, char *input_filen
       printf (msgcat_message (MSGCAT_CATALOG_UTILS, MSGCAT_UTIL_SET_COMPACTDB, COMPACTDB_MSG_PASS2));
     }
   status =
-    do_reclaim_addresses ((const OID ** const) class_oids, num_classes, &num_classes_fully_compacted, verbose_flag,
+    do_reclaim_addresses ((const OID **) class_oids, num_classes, &num_classes_fully_compacted, verbose_flag,
```

Comment:
Is cast necessary at all here?

Reply:
> Is cast necessary at all here?

Yes, otherwise: error: invalid conversion from ‘OID**’ to ‘const OID**’

Reply:
hm, I think the do_reclaim_addresses signature is actually incorrect. It probably should be `OID * const *class_oids`. What do you think?

Reply:
Yes, you're right